modules/ocf_xmpp/manifests/init.pp

```diff
@@ -0,0 +1,57 @@
+class ocf_xmpp {
+  # Needed to connect to MySQL
+  package { ['lua-dbi-mysql']: }
+
+  user { 'prosody':
+    groups  => 'ssl-cert',
+    require => [Package['prosody'], Package['ssl-cert']],
+  }
+
+  ocf::firewall::firewall46 {
+    '101 allow xmpp':
+      opts   => {
+        chain  => 'PUPPET-INPUT',
+        proto  => 'tcp',
+        dport  => [5222, 5269],
+        action => 'accept',
+      }
+  };
+
+  $vhost_name = $::host_env ? {
+    'dev'  => 'dev-xmpp.ocf.berkeley.edu',
+    'prod' => 'ocf.berkeley.edu',
```

Comment:
I think this should be `xmpp.ocf.berkeley.edu`?

Reply:
Yeah, this is correct